book/doctrine.rst

```diff
@@ -979,126 +1060,68 @@ To relate the ``Category`` and ``Product`` entities, start by creating a
             </entity>
         </doctrine-mapping>
 
-First, since a ``Category`` object will relate to many ``Product`` objects,
-a ``products`` array property is added to hold those ``Product`` objects.
-Again, this isn't done because Doctrine needs it, but instead because it
-makes sense in the application for each ``Category`` to hold an array of
-``Product`` objects.
+Doctrine does not *require* the "one" side of a one-to-many relationship to
+hold a collection of its "many" related entities. But in the context of our
+application, it makes sense for each ``Category`` object to hold a collection
+of ``Product`` objects. However, if we had decided against adding a ``$products``
+property to the ``Category`` class, then the ``Product`` entity's ``inversedBy``
+metadata would have to be omitted.
```

Comment:
I've tried to reword this a bit, but fear it may still be a little too clunky.  Feel free to have at it yourself during the merge :)